app/views/newbookings/index.html.erb

```diff
@@ -19,10 +21,8 @@
   <td><%= newbooking.pet.name %></td>
   <td><%= newbooking.pet.customer.address %></td>
   <td><%= newbooking.session %></td>
-  <td><%= link_to 'Destroy', newbooking_path(newbooking),
-               method: :delete,
-               data: { confirm: 'Are you sure?' } %>
-
+  <td><%= newbooking.service.name %></td>
```

Comment:
doing this can be a little expensive.

When you load `newbooking` the first time, it will make a database call.
When you call `newbooking.pet` or `newbooking.service` for the first time, it will make another database call to load the `pet` and the `service` for every `newbooking` we have. If there are 100 newbookings, then that will make 200 extra database calls.... This is the "N+1" problem if you want to read up about it.

Although it won't make much difference initially, if you scale up, that will make a lot of extra calls.

You can avoid this by eager loading - see https://guides.rubyonrails.org/active_record_querying.html#eager-loading-associations

Instead of doing something like `@newbooking = Newbooking.all` in the controller, you can do something like `@newbooking = Newbooking.all.includes(:pet, :service)` and that will collapse down to just 3 DB calls.

Something to try:
* create a load of bookings
* request this page and look at the logs; you should see some `SELECT ..` sql statements
* change the controller to eager load
* make the same request and look at the logs; you should see a lot fewer SQL